src/app/components/home-page/home-page.component.spec.ts

```diff
@@ -37,31 +37,31 @@ describe('HomePageComponent', () => {
     });
 
     it('should render total endpoint iframe first', () => {
-      expect(iframeElements[0].src).toEqual(component.totalEndpoints);
+      expect(iframeElements[0].src).toEqual(component.getUrl(component.totalEndpoints));
     });
 
     it('should render success iframe second', () => {
-      expect(iframeElements[1].src).toEqual(component.success);
+      expect(iframeElements[1].src).toEqual(component.getUrl(component.success));
     });
 
     it('should render unsuccessful iframe third', () => {
-      expect(iframeElements[2].src).toEqual(component.unsuccessful);
+      expect(iframeElements[2].src).toEqual(component.getUrl(component.unsuccessful));
     });
 
     it('should render unresponsive iframe fourth', () => {
-      expect(iframeElements[3].src).toEqual(component.unresponsive);
+      expect(iframeElements[3].src).toEqual(component.getUrl(component.unresponsive));
     });
 
-    it('should render new endpoints iframe fifth', () => {
-      expect(iframeElements[4].src).toEqual(component.newEndpoints);
-    });
+    // it('should render new endpoints iframe fifth', () => {
+    //   expect(iframeElements[4].src).toEqual(component.getUrl(component.newEndpoints));
```

Comment:
Why was this test commented out?

Reply:
nvm I think I see why, because  `// public newEndpoints = '&panelId=20';`

Reply:
yeah i just commented out the new endpoints vis since it's currently just a placeholder and throws a bunch of errors since it doesn't actually work.

Reply:
Followup question, if the `newEndpoints` viz is not in the works can we just delete the panelID definition and the test? Or atleast TODO them?

Reply:
sure, i can do that!